airflow/providers/amazon/provider.yaml

```diff
@@ -263,6 +264,7 @@ sensors:
   - integration-name: Amazon EC2
     python-modules:
       - airflow.providers.amazon.aws.sensors.ec2_instance_state
+      - airflow.providers.amazon.aws.sensors.ec2
```

Comment:
Shouldn't we remove the line above this? or do we need to keep it until we remove it entirely?

Reply:
cc @potiuk

Reply:
Removing it will cause failures for `verify_provider_package`.
pre-commits checks for comptability between yaml and modules found in the provider package.
removing the entries should be done when we remove the modules

Reply:
We have several of similar cases. For example:
https://github.com/apache/airflow/blob/fa96b093952f96449d6d328a2b9e9300b81cf08e/airflow/providers/amazon/provider.yaml#L312-L314